.github/workflows/test.yaml

```diff
@@ -275,6 +275,12 @@ jobs:
           --health-interval=5s
           --health-timeout=3s
           --health-start-period=10s
+      azurite:
+        image: mcr.microsoft.com/azure-storage/azurite
+        ports:
+          - "10000:10000"
+          - "10001:10001"
+          - "10002:10002"
```

Comment:
this is because you're only testing azure atm, right?  Will this need to extend to the other clouds too?

Reply:
The other services are implemented with a mock. Unfortunately there is not an equivalent for s3 and gcp